### ISSUE TITLE
Bootstrap theme - Content of the Meta tag "Description" shows the default text along the custom one in profiles page

### ISSUE THREAD

**Issue by optiroot** [moxi9/phpfox#795]
If you set a custom Site Description, it works along all the site in the HTML, and the code is:
`<meta name="description" content="example"/>`
It's okay, because in "content" it shows the text that you've set in PHPfox settings.

But if you go in a user profile page, the content of the tag is built mixing the custom description and the defaut description.

For example at me it shows:
`<meta name="description" content="Utente è sul sito. Site Name gives people the power to share and makes the world more open and connected. L'utente ha 33 amici. Registrati sul sito e connettiti con l'utente, messaggia all'utente o aggiungi l'utente come tuo amico."/>`

As you can see, it's mixed.

I use a Language Package made by me in Italian.
The phrase is translated!

This happens only if you are on a user profile page, otherwise the description is correct with the custom text.

**Comment by Data66PF** [comment]
This is a leftover from old settings we removed but really should restore for SEO purposes. We used to have the settings for what to put for the SEO stuff for profiles, blogs, etc. Since we don't have that now, we need to fix what is showing on the profiles or restore the settings we removed. I believe clients would prefer to restore the seo settings we used to have.

Setting used to be admincp/setting/edit/group-id_general/
core.meta_description_profile
Meta Description Profile
(General) 
core.meta_description_profile
This is the meta description provided on a users profile that is included with their personal information. It is advised for this to not be too long in order to leave room for the users personal information.

and the default phrase for that is:
Site Name gives people the power to share and makes the world more open and connected.
